src/test/regress/spec/isolation_metadata_sync_vs_all.spec

```diff
@@ -0,0 +1,205 @@
+// the test expects to have zero nodes in pg_dist_node at the beginning
+// add 2 nodes and enable metadata syncing to a single node.
+setup
+{
+	SET citus.shard_replication_factor to 1;
+
+	ALTER SEQUENCE pg_catalog.pg_dist_colocationid_seq RESTART 123000;
+	ALTER SEQUENCE pg_catalog.pg_dist_groupid_seq RESTART 123000;
+	ALTER SEQUENCE pg_catalog.pg_dist_node_nodeid_seq RESTART 123000;
+	ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 123000;
+
+	SELECT 1 FROM master_add_node('localhost', 57637);
```

Comment:
why do we add / remove nodes in the setup / tear down? Are they strictly necessary?

Reply:
No they are not. I used to have one node with metadata, and one without. This is no longer the case now.

My original plan was to test `citus_add_node`, `start_metadata_sync_to_node`, `trigger_metadata_sync` and I no longer need to add and drop the nodes at every permutation.